chameleonultragui/lib/gui/page/flashing.dart

```diff
@@ -11,15 +11,14 @@ class FlashingPage extends StatelessWidget {
     var appState = context.watch<MyAppState>();
 
     return Scaffold(
-        appBar: AppBar(
-          title: const Text('Chameleon DFU'),
```

Comment:
Why removed?

Reply:
I have no idea? Not intentional, guess some merge issue. WIll add it back